improve docs for datadog integration (#26)
flightcontrolhq/website: motif-docs/pages/guides/datadog/support-for-fargate.mdx

  "minInstances": 1,
  "maxInstances": 1,
  "dockerLabels": {
    "com.datadoghq.ad.instances": "[{\"host\": \"%%host%%\", \"port\": 3000}]", //change the port if needed
    "com.datadoghq.ad.check_names": "[\"DISPLAY_NAME\"]", // name that will show in datadog
    "com.datadoghq.ad.init_configs": "[{}]"
  },
  "experimental": { // make sure the datadog object is added under experimental
    "datadog": {
      "enabled": true,
      "datadogSite": "us5.datadoghq.com", //plaintext

>       "datadogApiKey": "<REFRENCE_TO_DD_API_KEY_IN_AWS>" //from parameter store or secrets manager

Reviewer: ```suggestion
      "datadogApiKey": "<REFRENCE_TO_DD_API_KEY_IN_AWS>" //from parameter store, needs to be manually added
```
